### PR TITLE
Replace the MP in editor part costs with a circle-less MP icon

### LINKED ISSUE #3589: Replace the word MP in the editor cost texts with the MP icon
Like this:
replaced mp icon

though there was also a request that maybe the icon should be without the circle around it to make the editor less busy visually, so I'll also tag this as 2d art.

### PR DESCRIPTION
**Pull request by Kasterisk**
**Brief Description of What This PR Does**

<image src="https://user-images.githubusercontent.com/54026083/185777717-8e6010fc-72da-4117-bb5b-fc9573250885.png" width=40%>

They now uses a circle-less MP icon. Looks tolerable to me.

**Related Issues**

Closes #3589 

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by nickthenick** [comment]
Looks good. Only thing is that I realized it looks off with the text not centered. Is there a way to center the text, and then have the icon appear to the left of it? 

If the icon and text are all one element, perhaps a way to do it would be to add empty space at the end of the element equal to the space occupied by the icon in front so that overall the text gets centered.

**Comment by Kasterisk** [comment]
I've now made it centered. And yeah a simple spacer Control does the job well.

File: src/microbe_stage/editor/MicrobePartSelection.cs [hunk 1]
```diff
@@ -136,17 +137,30 @@ public override void _ExitTree()
...
+            vbox.AddConstantOverride("separation", 4);
+        }
     }
 
     private void UpdateLabels()
     {
         if (mpLabel == null || nameLabel == null)
             return;
 
-        mpLabel.Text = TranslationServer.Translate("MP_COST").FormatSafe(MPCost);
```
Comment (hhyyrylainen) [inline src/microbe_stage/editor/MicrobePartSelection.cs:148]: When investigating this change I noticed that there are multiple other places still in the game where the MP cost is not shown with the icon...

**Review by hhyyrylainen — APPROVED** [review]
I did some cleanup to make the code good. I'll merge this now as it seems this likely won't get worked on more for now even though there's now a disconnect between a few places that still use the word MP instead of the MP icon to show the cost.

**Comment by hhyyrylainen** [comment]
Follow up issue: https://github.com/Revolutionary-Games/Thrive/issues/3756

**Pull request merged**